recipe/build.sh

```diff
@@ -1,11 +1,17 @@
 set -e
 
+export OCL_ICD_VENDORDIR="${PREFIX}/etc/OpenCL/vendors"
+
+# Make sure `conda-build` keeps this directory.
+mkdir -p "${OCL_ICD_VENDORDIR}"
+touch "${OCL_ICD_VENDORDIR}/.conda_keep"
```

Comment:
As `conda-build` doesn't keep empty directories yet, we have to add a stub file to keep this directory here.

If this is going to be problematic, we could add `post-unlink.sh` and `pre-unlink.sh` scripts to create this directory. Though such scripts could just as easily be used to clean out the stub file. Really no preference on my end as long as the directory is there when the package is installed.

Alternatively we could put something useful in this directory like a README explaining what the purpose of this directory is and how it should be used.